lib/kitchen/templates/eob_section_template.xhtml.erb

```diff
@@ -0,0 +1,11 @@
+<% main_title_tag = @in_composite_chapter ? 'h3' : 'h2' %>
+<div class="os-eob os-<%=@container_class_type%>-container" data-type="composite-page" data-uuid-key="<%= @uuid_key %>">
+  <<%= main_title_tag %> data-type="<%= @in_composite_chapter ? 'title' : 'document-title' %>">
+    <span class="os-text"><%= @title %></span>
+  </<%= main_title_tag %>>
+  <div data-type="metadata" style="display: none;">
+    <h1 data-type="document-title" itemprop="name"><%= @title %></h1>
+    <%= @metadata.paste %>
+  </div>
+  <%= @content %>
+</div>
```

Comment:
Actually there is already such template: https://github.com/openstax/kitchen/blob/b8aadf6e0bdff60f0b4fc3ee37fa23f81203c443/lib/kitchen/templates/eob_section_title_template.xhtml.erb . Maybe title is a little bit misleading.

First line from your template I think is not necessary in EOB. As I write in my previous comment, the only one composite chapter is Answer Key. Even we decide to use this for Answer Key composite pages I think they also have `h2` headers, not `h3`.